worker-message-prioritization-distribution/src/main/java/com/github/workerframework/workermessageprioritization/redistribution/lowlevel/StagingQueueTargetQueuePair.java

```diff
@@ -153,7 +153,6 @@ public void handleStagedMessage(final String consumerTag, final Envelope envelop
         AMQP.BasicProperties basicProperties = new AMQP.BasicProperties.Builder()
                 .contentType(properties.getContentType())
                 .deliveryMode(properties.getDeliveryMode())
-                .priority(properties.getPriority())
```

Comment:
@andyreidz unsure about this, feels like it should still be removed if this is a back up for the staging queues created by the target queue channel, but perhaps if we are merging these changes before actually switching to quorum queues, it should be left in.

Reply:
Removing this as the ability to set priority has now been removed in worker-framework